components/org.wso2.carbon.identity.api.server.common/src/main/java/org/wso2/carbon/identity/api/server/common/Util.java

```diff
@@ -100,22 +101,32 @@ public static String base64URLDecode(String value) {
 
     /**
      * Build 'next' and 'previous' pagination links.
-     * @param limit Value of the 'limit' parameter.
-     * @param currentOffset Value of the 'currentOffset' parameter.
+     *
+     * @param limit                  Value of the 'limit' parameter.
+     * @param currentOffset          Value of the 'currentOffset' parameter.
      * @param totalResultsFromSearch Value of the 'totalResultsFromSearch' parameter.
-     * @param servicePathComponent API service path. E.g: applications/
+     * @param servicePathComponent   API service path. E.g: applications/
+     * @param requiredAttributes     Value of the 'attributes' parameter.
+     * @param filter                 Value of the 'filter' parameter.
      * @return A map containing pagination link key-value pairs.
      */
     public static Map<String, String> buildPaginationLinks(int limit, int currentOffset, int totalResultsFromSearch,
```

Comment:
Let's not change the public method. Instead, we can deprecate this and introduce a new method.

Reply:
Fixed with https://github.com/wso2/identity-api-server/pull/501/commits/8ea930153ca5949324919cb611b7b9e99b894e6e